web/pages/semantics.md

```diff
@@ -0,0 +1,64 @@
+# Semantics in K
+
+<br>
+
+## Featured
+
+* **[C semantics (Jul 2010 - Present)](https://github.com/kframework/c-semantics)**
+
+  The K semantics of the [C programming language](<https://en.wikipedia.org/wiki/C_(programming_language)>) specifies the translation, linking, and execution semantics of the C language according to the official C standard. It has been used to build tools like RV-Match, which detects undefined behaviors in users programs by running their test-suites through the C semantics.
+
+* **[KWasm (Aug 2015 - Present)](https://github.com/kframework/wasm-semantics)**
+
+  KWasm is the K semantics of [WebAssembly](https://webassembly.github.io/spec/). WebAssembly is a low-level (but simple and streamlined) assembly language that was originally developed to provide a fast execution engine for browser-based tools. More recently, it has been used in several blockchain smart-contract platforms as the underlying language for executing financial agreements. KWasm has been used for measuring coverage of test-suites over Wasm code and verifying programs which are compiled to Wasm.
+
+* **[Elrond (Aug 2015 - Present)](https://github.com/runtimeverification/elrond-semantics)**
+
+  The semantics of [Elrond](https://elrond.com/) is a wrapper around KWasm which extends it with the Elrond blockchain primitives. This extension enables developers targeting the Elrond blockchain to measure coverage of their test-suite directly at the Wasm level.
```

Comment:
@hjorthjort please review these paragraphs.

Reply:
Text looks good. The dates for Elrond should be July 2020 - Present.